src/components/elementView/elementTabContainer.tsx

```diff
@@ -95,10 +95,50 @@ class ElementTabContainer extends React.Component<
             />
           }
         />
+
         <Tab
-          className="tab do-not-print"
-          id="assets"
+          className="tab tab-header do-not-print"
+          id="assets-header"
           title="Assets"
+          disabled={true}
+        />
+
+        <Tab
+          className="tab-sub do-not-print"
+          id="assets"
+          title="Racked Assets"
+          panel={
+            <ElementTab
+              datacenters={this.state.datacenters}
+              currDatacenter={this.state.currDatacenter}
+              onDatacenterSelect={this.onDatacenterSelect}
+              {...this.props}
+              element={ElementType.ASSET}
+              isActive={false}
+            />
+          }
+        />
+
+        <Tab
+          className="tab-sub do-not-print"
+          id="offline-storage-assets"
+          title="Offline Storage Assets"
+          panel={
+            <ElementTab
+              datacenters={this.state.datacenters}
+              currDatacenter={this.state.currDatacenter}
+              onDatacenterSelect={this.onDatacenterSelect}
+              {...this.props}
+              element={ElementType.ASSET}
+              isActive={false}
+            />
+          }
+        />
+
+        <Tab
+          className="tab-sub do-not-print"
+          id="decommissioned-assets"
```

Comment:
same as above - this route doesn't exist yet, so page won't load when clicked on (future PR)